### DISCUSSION TITLE
OpenWebUI always crashes, when pulling some web pages

### DISCUSSION THREAD

**Discussion by SlavikCA** [open-webui/open-webui#8120] [Issues]
## Installation Method

OpenWebUI installed in Kubernetes cluster, Ollama runs on VM, which has RTX 3090.

## Environment

- **Open WebUI Version:** 0.5.1
- **Ollama (if applicable):** 0.5.4
- **Browser (if applicable):** Chrome
- Model used: QWEN2.5:32b

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Actual Behavior:

- start new chat
- type: `#https://www.soyoustart.com/us/essential-servers/`
- OpenWebUI crashes

## Logs and Screenshots

**Docker Container Logs:**

```
...
INFO  [open_webui.routers.retrieval] save_docs_to_vector_db: document Low-cost Dedicated Servers - Servers at an unbeatable price 3eaecae2725f3edb83ae9b483a0da9313255bcfeeac154444a5ea6e3f80f09d
INFO  [open_webui.routers.retrieval] collection 3eaecae2725f3edb83ae9b483a0da9313255bcfeeac154444a5ea6e3f80f09d already exists
INFO  [open_webui.routers.retrieval] deleting existing collection 3eaecae2725f3edb83ae9b483a0da9313255bcfeeac154444a5ea6e3f80f09d
INFO  [open_webui.routers.retrieval] adding to collection 3eaecae2725f3edb83ae9b483a0da9313255bcfeeac154444a5ea6e3f80f09d
```

## Additional info

Other pages / links works ok. But on this link it always crashes.

**Comment by tjbck** [comment]
![image](https://github.com/user-attachments/assets/5aaecd4a-68ff-46a6-bea2-5799ce4f4e3a)

Works on my end, keep us updated!
